src/Playwright.Tests/BaseTests/PlaywrightAssert.cs

```diff
@@ -48,7 +46,5 @@ internal static class PlaywrightAssert
                 return t;
             }
         }
-
-        internal static void DebugLog(string text) => TestContext.Progress.WriteLine(text);
```

Comment:
Let's keep this one please, we use it for debugging.

Reply:
done